src/command_modules/azure-cli-acr/azure/cli/command_modules/acr/_validators.py

```diff
@@ -0,0 +1,96 @@
+#---------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+#---------------------------------------------------------------------------------------------
+
+import re
+import uuid
+
+from azure.cli.command_modules.acr.mgmt_acr.models import RegistryNameCheckRequest
+
+from azure.cli.core._util import CLIError
+from azure.cli.command_modules.storage._factory import storage_client_factory
+
+from ._constants import (
+    RESOURCE_TYPE,
+    ALLOWED_ROLES
+)
+from ._factory import (
+    get_acr_service_client,
+    get_arm_service_client
+)
+from ._arm_utils import arm_get_storage_account_by_name
+
+import azure.cli.core._logging as _logging
+logger = _logging.get_az_logger(__name__)
+
+def validate_registry_name(namespace):
+    if namespace.registry_name:
+        registry_name = namespace.registry_name
+        if len(registry_name) < 5 or len(registry_name) > 60:
+            raise CLIError('The registry name must be between 5 and 60 characters.')
+
+        p = re.compile('^([A-Za-z0-9]+)$')
+
+        if not p.match(registry_name):
+            raise CLIError('The registry name can contain only letters and numbers.')
+
+def validate_registry_name_create(namespace):
+    if namespace.registry_name:
+        client = get_acr_service_client()
+
+        result = client.operation.check_name_availability(
+            RegistryNameCheckRequest(
+                namespace.registry_name,
+                RESOURCE_TYPE
+            )
+        )
+
+        if not result.name_available: #pylint: disable=E1101
```

Comment:
We use the pylint error names instead of codes so it's clear when reading the code what is being disabled. So this would be `# pylint: disable=no-member`